src/counts/embezzler.ts

```diff
@@ -0,0 +1,216 @@
+import { canAdventure, getClanLounge, itemAmount } from "kolmafia";
+import {
+  $effect,
+  $familiar,
+  $item,
+  $location,
+  $locations,
+  $monster,
+  $skill,
+  ChateauMantegna,
+  CombatLoversLocket,
+  Counter,
+  CrystalBall,
+  get,
+  have,
+  property,
+  SourceTerminal,
+  sum,
+} from "libram";
+
+const embezzler = $monster`Knob Goblin Embezzler`;
+
+export function embezzlerCount(): number {
+  return sum(embezzlerCounts, (countEntry: NameCountPair) => countEntry[1]());
+}
+
+interface CountFunction {
+  (): number;
+}
+
+type NameCountPair = [string, CountFunction];
```

Comment:
```suggestion
type NameCountPair = [string, () => number];
```

that being said, I think this would be easier and more legible as an object type instead of as a tuple, so
```
type EmbezzlerCount = { name: string, quantity: () => number };
```